gwsurrogate/new/precessing_surrogate.py

```diff
@@ -1094,3 +1135,197 @@ def __call__(self, x, fM_low=None, fM_ref=None, dtM=None,
             dynamics = None
 
         return timesM, h, dynamics
+
+class PrecessingSurrogateMultiDomain(object):
```

Comment:
Move this into a separate file to avoid bloat?

Reply:
I know @sfield17 earlier said we're going to have a refactoring PR to break classes out. We could do this in the big refactoring PR

Reply:
Agree, happy to leave it to Scott.